src/api/contributionsApi.testData.ts

```diff
@@ -0,0 +1,70 @@
+import { EpicTests } from '../lib/variants';
+
+export const configuredTests: EpicTests = {
```

Comment:
Not major, but it might be worth raising this bit as a separate PR as it's relatively isolated and would make it easier to review things separately.

Reply:
Oh. do you mean add the test data separately, but unused? (I think the only tests which use this data are newly added in this PR).

Reply:
yeah or as a second stage for example

Reply:
Cool - pulled out into a separate PR: #94. I'll rebase this branch once that's merged which will remove it from the diff.